### ISSUE TITLE
The Invoice History in the Payment Dashboard doesn't show the history 

### ISSUE THREAD

**Issue by lauriebergeron** [narcitymedia/lilium-cms#432]
**Describe the bug**
When I go on the Payment Dashboard and click on Invoice History, I see the last payments made on June 28. Then, when I click on Load More, it shows June 28 again and again. Seems like it's an infinite scroll of June 28. 

**To Reproduce**
1. Go to the Payment Dashboard on Lilium
2. Click on click on Invoice History
3. Scroll down to Load More 

**Expected behavior**
I need to have access to the past data of all payments made to contractors. Payment is made every other week, and I would need access to all of them since the invention of the Payment Dashboard for internal revenue documents. 

**Section**
Payment Dashboard

**Comment by lauriebergeron** [comment]
When I scroll now, it's June 28, then June 14, and then an infinite scroll of June 28, June 14, June 28, June 14

**Comment by lauriebergeron** [comment]
Don't know if you touched something, but I am now able to see the history! 

Also! When I try researching a name of an author in the Search bar, it says _Nothing was found using the current filters._ Would it be possible to research certain authors to know how much we paid them? Would help facilitate the reports!

**Comment by Gaboik** [comment]
> Don't know if you touched something, but I am now able to see the history!
> 
> Also! When I try researching a name of an author in the Search bar, it says _Nothing was found using the current filters._ Would it be possible to research certain authors to know how much we paid them? Would help facilitate the reports!

@lauriebergeron there is the **Payment History** section for that. It's accessible from the left menu just below the **Payment Dashboard** section. It won't allow you to search but you can generate a payment report by contractor

**Comment by Gaboik** [comment]
The filter box in the Invoice History tab of the **Payment Dashboard** section is indeed not working proprely

**Comment by lauriebergeron** [comment]
Oh wow! I was looking at Payment History inside the Payment Dashboard. I didn't know there was another Payment History tab! Sharp! Just did everything manually hahaha but this will save SO MUCH time! Would be cool to have a Search Bar to be able to find a contractor. I am using Command F for now, which works fine.

**Comment by Gaboik** [comment]
> Would be cool to have a Search Bar to be able to find a contractor
Yeah I agree, this could easily be implemented on the front end only since it would only search trough the downloaded report

**Comment by Gaboik** [comment]
This is still happening in production

**Comment by lauriebergeron** [comment]
This is still not working!
https://www.loom.com/share/3dbe1d5118754e7bb927f4995051460c

**Comment by lauriebergeron** [comment]
It's the same dates over and over again! My workaround is to play with it for a few minutes, close it, open it again, etc. until it works.
